webpack/JobWizard/JobWizard.js

```diff
@@ -9,23 +9,35 @@ import { AdvancedFields } from './steps/AdvancedFields/AdvancedFields';
 import { JOB_TEMPLATE } from './JobWizardConstants';
 import { selectTemplateError } from './JobWizardSelectors';
 import Schedule from './steps/Schedule/';
+import HostsAndInputs from './steps/HostsAndInputs/';
 import './JobWizard.scss';
 
 export const JobWizard = () => {
   const [jobTemplateID, setJobTemplateID] = useState(null);
   const [category, setCategory] = useState('');
+
   const [advancedValues, setAdvancedValues] = useState({});
+  const [templateValues, setTemplateValues] = useState({}); // TODO use templateValues in advanced fields - description
```

Comment:
is there a redmine for the todo?

Reply:
https://github.com/theforeman/foreman_remote_execution/pull/605